client-v2/src/components/FrontsEdit/FrontDetailView.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import ArticleFragmentForm from './ArticleFragmentForm';
+import FrontCollectionsOverview from './FrontCollectionsOverview';
+import { connect } from 'react-redux';
+import {
+  ArticleFragmentMeta,
+  CollectionItemSets
+} from 'shared/types/Collection';
+import { updateArticleFragmentMeta as updateArticleFragmentMetaAction } from 'actions/ArticleFragments';
+import {
+  editorClearArticleFragmentSelection,
+  selectEditorArticleFragment
+} from 'bundles/frontsUIBundle';
+import { Dispatch } from 'types/Store';
+import { State } from 'types/State';
+
+interface ContainerProps {
+  id: string;
+  browsingStage: CollectionItemSets;
+}
+
+interface ComponentProps extends ContainerProps {
+  selectedArticleFragment: { id: string; isSupporting: boolean } | void;
+  updateArticleFragmentMeta: (id: string, meta: ArticleFragmentMeta) => void;
+  clearArticleFragmentSelection: (id: string) => void;
+}
+
+const FrontsDetailView = ({
+  selectedArticleFragment,
+  id,
+  browsingStage,
+  clearArticleFragmentSelection,
+  updateArticleFragmentMeta
```

Comment:
I'd made a mistake here in the previous PR -- I hadn't included updateArticleFragmentMeta as a destructured prop, and as a result the component was calling the action creator, which has the same name. This meant that nothing was dispatched when onSave was called 😢

Reply:
Oh wow that's fiddly ...

Reply:
Is the signature exactly the same too?

Reply:
yuh-huh 👎 Potential way of spotting these things in future -- adopt naming convention for actions, e.g. actionUpdateArticleFragmentMeta